src/cmd/new-pull-request.go

```diff
@@ -50,7 +50,7 @@ Example: your SSH identity should be something like
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 		if git.IsOffline() {
-			util.ExitWithErrorMessage("Error: cannot create new pull requests in offline mode.")
+			util.ExitBecauseOffline("new-pull-request")
 		}
```

Comment:
Sorry missed this in the previous PR but I think we should return an error to PreRunE instead of exiting ourselves. This could be something like
```go
err := git.ValidateIsOnline()
if err != nil {
  return err
}
```
I also think this check should be at the end instead of the beginning. The error message could be updated to "This command requires an active internet connection". Using `PreRunE` it will print the usage below the error